project-base/app/deploy/deploy-project.sh

```diff
@@ -108,8 +113,15 @@ function deploy() {
 }
 
 function merge() {
+    # Specify consumers configuration with the default configuration in the format:
+    # <consumer-name>:<transport-names-separated-by-space>:<number-of-consumers>
+    DEFAULT_CONSUMERS=(
+        "example:example_transport:1"
```

Comment:
so I need to put my consumers here manually, right? Is not there a way on how to generate this from the `messeger.yaml` config? :innocent:  And when I need to upscale a particular consumer, I just increase the number here and run deploy, am I right?